operations/finance.md

```diff
@@ -440,17 +469,90 @@ For most amounts, precision of values depends on `TaxPrecision` of [Enterprise](
 | `Code` | number | required | Code corresponding to tax type. |
 | `Value` | number | required | Amount of tax applied. |
 
-#### Credit card transaction
+#### Revenue type
+
+* `Service`
+* `Product`
+* `Additional`
+
+#### Order item data
 
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
-| `Id` | string | required | Unique identifier of the credit card transaction. |
-| `PaymentId` | string | required | Unique identifier of the [Accounting item](#accounting-item). |
-| `ChargedAmount` | [Amount](finance.md#amount-value) | required | Charged amount of the transaction. |
-| `SettledAmount` | [Amount](finance.md#amount-value) | optional | Settled amount of the transaction. |
-| `Fee` | [Amount](finance.md#amount-value) | optional | Fee of the transaction. |
-| `SettlementId` | string | optional | Identifier of the settlement. |
-| `SettledUtc` | string | optional | Settlement date and time in UTC timezone in ISO 8601 format. |
+| `Discriminator` | string [Order item data discriminator](#order-item-data-discriminator) | required | Type of the order item (e.g. `ProductOrder`). |
+| `Value` | object | optional | Based on order item discriminator, e.g. [Product order item data](#product-order-item-data) or `null` for types without any additional data.  |
+
+#### Order item data discriminator
```

Comment:
shouldn't this be aligned with the serializer?
![image](https://user-images.githubusercontent.com/7511272/133613555-64fdb5e0-d450-4a23-9a16-61923e048ae7.png)

Reply:
The one you posted is from Navigator, damn APIs looking all similar 😄 
![image](https://user-images.githubusercontent.com/1150547/133623573-f18038e8-d1ea-46a2-a663-4108b87041da.png)